docs/sdk/server-side/node-js/index.md

```diff
@@ -4,8 +4,210 @@ sidebar_position: 1
 slug: /sdk/server-side/node-js
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This category contains topics explaining how to configure Bucketeer's Node.js SDK.
+
 ## Getting started
 
 Before starting, ensure that you follow the [Getting Started](/getting-started) guide.
 
-Coming soon.
+### Installing dependency
+
+Install the dependency in your application.
+
+<Tabs>
+<TabItem value="npm" label="npm">
+
+```sh showLineNumbers
+npm install @bucketeer/node-server-sdk
+```
+
+</TabItem>
+<TabItem value="yarn" label="Yarn">
+
+```sh showLineNumbers
+yarn add @bucketeer/node-server-sdk
+```
+
+</TabItem>
+</Tabs>
+
+### Importing SDK
+
+Import the Bucketeer SDK into your application code.
+
+<Tabs>
+<TabItem value="js" label="JavaScript">
+
+```js showLineNumbers
+import { initialize } from '@bucketeer/node-server-sdk';
+```
+
+</TabItem>
+</Tabs>
+
+### Configuring client
+
+Configure the SDK config and user configuration.
+
+<Tabs>
+<TabItem value="js" label="JavaScript">
+
+```js showLineNumbers
+const config = {
+  host: 'YOUR_API_URL',
+  token: 'YOUR_API_KEY',
+  tag: 'YOUR_TAG',
+};
+```
+
+</TabItem>
+</Tabs>
+
+:::info Custom configuration
+
+Depending on your use, you may want to change the optional configurations available.
+
+- **pollingIntervalForRegisterEvents** (Default is 1 minutes)
+- **logger** (Default is `logger.DefaultLogger`)
+
+:::
+
+### Initializing client
+
+Initialize the client by passing the configurations in the previous step.
+
+<Tabs>
+<TabItem value="js" label="JavaScript">
+
+```js showLineNumbers
+const client = initialize(config);
+```
+
+</TabItem>
+</Tabs>
+
+If you want to use the feature flag on Splash or Main views, and the user opens your application for the first time, it may not have enough time to fetch the variations from the Bucketeer server.
```

Comment:
Because the SDK doesn't cache the data and always gets the latest variation, we don't need this whole section.

https://github.com/bucketeer-io/bucketeer-docs/pull/42/files#diff-60b214ad180a48d85cdb00d5b203f47181c31e2447de70e83488d06701dbea45R92-R108